custom-vu/src/main/kotlin/jces1209/vu/action/CreateAnIssue.kt

```diff
@@ -49,10 +49,22 @@ class CreateAnIssue(
         val driver = jira.driver
         driver
             .wait(
-                condition = elementToBeClickable(createIssueButton),
+                condition = or(*createIssueButtons.map { elementToBeClickable(it) }.toTypedArray()),
                 timeout = Duration.ofSeconds(10)
             )
-            .click()
+        createIssueButtons
+            .forEach(fun(locator) {
+                val foundButtons = driver.findElements(locator)
+                if (!foundButtons.isEmpty()) {
+                    foundButtons
+                        .forEach {
```

Comment:
`forEach` will do nothing if `foundButtons.isEmpty()`, so we don't have to `if` it